### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by whippingdot** [Repl-it-Coders/Repl.it-2020#1] [Ideas]
Hey guys, write your ideas for the website on this discussion. I will change it into an issue if it is a good idea.

**Comment by Isaiah-Day** [comment]
We should definitely incorporate a bunch of languages, I've got some python, HTML, CSS, some JS is soon to come, any suggestions?

**Reply by whippingdot** [reply]
All I was thinking was a website. I don't know how the flask/django will fit in with the html, css, and js but I hope it works. I am learning CSS and JS rn so I wud suggest letting me do the HTML. What are you good at?

**Reply by ghost** [reply]
@whippingdot @Isaiah08-D 
what do you want me to build in the python part? Also, how do I edit it?

**Reply by whippingdot** [reply]
Hey, go to repl.it and create a new project, but instead of naming it or choosing any language, press the github tab thing:
![image](https://storage.googleapis.com/replit/images/1608181284283_47acc96a0a608b2c691ed7c134f55d6e.png)

There paste the url of this repository, which is https://github.com/Repl-it-Coders/Repl.it-2020/. After that, press the import from github button and then you can edit the python. Just know that the python part is flask, and you have to know flask to do it. If you don't know flask, then you can do the HTML with me. I am not free now, but I will be free in some time.

**Reply by ghost** [reply]
thx!

**Reply by ghost** [reply]
what is the flask?

**Reply by whippingdot** [reply]
flask is something in Python, it is used to build websites. If you don't know flask or Django then do the HTML part. Do you know HTML? If you don't then wait for me, I will be free in 2 hrs.

**Reply by ghost** [reply]
Bro, I am sorry, but I kinda know HTML. But, I have to go to bed. You can start it without me, and I will work on it during school tomorrow. Also, in repl.it, you can share repls to allow other people to edit it. You might want to do that so that we can work together.

**Reply by whippingdot** [reply]
Ok, will do. It is ok that you have to go to bed, it does seem late for you(cause it seems that you are in USA). You go to bed, and I will do the most I can do. I will try to create comments in the code telling you what to do. Thanks for all you help anyway. GN!

**Reply by Isaiah-Day** [reply]
All the html is the templates folder and the css is in the static folder. The readme has a description of all the files.

**Reply by whippingdot** [reply]
I know, I saw the folders. 

#### On a different note
Add a profile pic, change your profile. Add a bio and that stuff. Your profile is empty.

**Reply by Siddharth816** [reply]
> All I was thinking was a website. I don't know how the flask/django will fit in with the html, css, and js but I hope it works. I am learning CSS and JS rn so I wud suggest letting me do the HTML. What are you good at?

dude I dont know about JS but django can surely fit in HTML you should really take the advantage of it ....

**Reply by Siddharth816** [reply]
> Hey, go to repl.it and create a new project, but instead of naming it or choosing any language, press the github tab thing:
> ![image](https://camo.githubusercontent.com/b3feaa5533e14ee3cace6c7f9884657b7240276768150f24e5cf8168f1433416/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f7265706c69742f696d616765732f313630383138313238343238335f34376163633936613061363038623263363931656437633133346635356436652e706e67)
> 
> There paste the url of this repository, which is https://github.com/Repl-it-Coders/Repl.it-2020/. After that, press the import from github button and then you can edit the python. Just know that the python part is flask, and you have to know flask to do it. If you don't know flask, then you can do the HTML with me. I am not free now, but I will be free in some time.

IDK it is not working with me

**Reply by Siddharth816** [reply]
> @whippingdot @Isaiah08-D
> what do you want me to build in the python part? Also, how do I edit it?

read the guide that whipping dot sent

**Reply by whippingdot** [reply]
No, don't do any of this. This is all old stuff @Siddharth816. Just join the repl I invited you to. We have already done lots in the project. The last thing we need to do is make the suggestions work and finish the timeline. We are using Node.js for the routing.

**Reply by firefish111** [reply]
`gh repo clone Repl-it-Coders/Repl.it-2020`

**Reply by whippingdot** [reply]
what @firefish111 ??? We are already using a repl, don't clone it unless you are not using repl.it. All of us are in a repl I will invite you.

**Comment by Siddharth816** [comment]
hi guys I just joined ...... wanted to wish u all a very happy new year in advance ....

**Reply by whippingdot** [reply]
Thanks

**Comment by Siddharth816** [comment]
![image](https://user-images.githubusercontent.com/76802411/103395296-9ccd4880-4b53-11eb-83d6-c32e44aa2452.png)

now i see this is'nt peice of cake to find all the repl's and posts that trended  the most this year (can be complicated) we will surely need something like an algorithm ..... (my suggestion) or else we are sure to run into errors 🙂🙂

**Reply by whippingdot** [reply]
NO, WHAT ARE YOU DOING. Join the repl I invite you to. Do NOT clone this. I already have. Please do not. Stop, join the repl I invite you to.

**Reply by whippingdot** [reply]
And @Siddharth816 i finished the timeline. I have gotten all 88 announcements just now. I am adding them to the repo. I did it manually and I am done. Please stop. Sorry I might be seeming harsh but that is the old version And I don't want you to commit something. Delete that repl and join the one I invited you to.

**Comment by Siddharth816** [comment]
can I get  a simple link to the Repl plz .....@whippingdot

**Reply by whippingdot** [reply]
https://repl.it/@Whippingdot/Replit-2020/
